src/command/setup.rs

```diff
@@ -165,7 +165,7 @@ mod os {
             command.arg("Path");
             command.arg(format!("{};{}", shim_dir, path));
 
-            debug!("Modifying User Path with command: {:?}", command);
+            debug!("Modifying Default Path with command: {:?}", command);
```

Comment:
This should also be User Path, as it refers to the User Path environment variable, as opposed to the System Path variable in Windows.

Reply:
👍